src/awkward/_v2/contents/unionarray.py

```diff
@@ -81,6 +81,17 @@ def contents(self):
     def nplike(self):
         return self._tags.nplike
 
+    @property
+    def numcontents(self):
+        return len(self._contents)
```

Comment:
This is another method that doesn't need to be translated. The testing code can be changed from `array.numcontents` to `len(array.contents)`. Many of these were in the old version because C++ doesn't have all the conveniences that Python has.